packages/js-drive/test/unit/identity/masternode/handleUpdatedScriptPayoutFactory.spec.js

```diff
@@ -68,7 +68,7 @@ describe('handleUpdatedScriptPayoutFactory', () => {
     const newPubKeyData = Buffer.alloc(20, '0');
     const identityPublicKeys = identity.getPublicKeys();
 
-    await handleUpdatedScriptPayout(
+    const result = await handleUpdatedScriptPayout(
```

Comment:
Let's make result format format canonical for new, update and remove methods. I would recommend to use a class with `merge` method